src/Administration/Resources/app/administration/src/app/component/grid/sw-pagination/sw-pagination.scss

```diff
@@ -90,15 +90,40 @@ $sw-pagination-border-radius: $border-radius-default;
     }
 
     .sw-pagination__per-page {
-        display: inline-block;
         font-size: $font-size-xs;
         line-height: 16px;
         position: absolute;
         right: 0;
+        top: calc(50% - 18px);
         margin: 0 20px;
 
         @media screen and (max-width: 900px) {
             display: none;
         }
+
+        .mt-select {
```

Comment:
Is this something we can port into the library? As it's scoped styles have the potential to break this styling 🤔

Reply:
In theory yes. But this specific styling adjustments are for the pagination. And we have a custom pagination also in the Meteor library. So it wouldn't make sense to support this case in the library